assets/views.py

```diff
@@ -189,3 +190,39 @@ def perform_destroy(self, instance):
         if instance.deleted_at is None:
             instance.deleted_at = now()
             instance.save()
+
+
+assets_stats_schema = ManualSchema(
+    fields=[],
+    description="Returns Assets stats: total number of assets, total number of assets completed,"
+                "total number of assets with personal data, and assets per department (total, "
+                "completed, with personal data)"
+)
+
+
+@api_view(['GET'])
+@renderer_classes((JSONRenderer, BrowsableAPIRenderer))
+@schema(assets_stats_schema)
```

Comment:
Shouldn't this be behind some permission? #78 suggests a new scope. Perhaps ``assetregister:metrics``?

Reply:
I'm unsure about that, we are going to make public all the data so I don't see why we should protect this.